src/components/ContactGroups/ContactGroupCreation.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import { Icon, Input, translate } from 'cozy-ui/react'
+import classNames from 'classnames'
+import './index.styl'
+
+const normalizeGroupData = name => {
+  return {
+    name: name,
+    metadata: {
+      version: 1
+    }
+  }
+}
+class ContactGroupCreation extends Component {
+  state = {
+    isInput: false,
```

Comment:
Can we find a better name for `isInput`? `isInputActive` or `isInputDisplayed` maybe?